### PR TITLE
Add ops

### PR DESCRIPTION
**Pull request by kaikai1024**
preview:

execption handle

2019-09-19-163727_835x937_scrot

service monitor

2019-09-19-163738_846x206_scrot

### CODE REVIEW

**Review by nanmuxiusi — DISMISSED** [review]

File: docs/cita/ops/exception-handle.md [hunk 1]
```diff
@@ -2,3 +2,47 @@
 id: exception-handle
 title: 异常排查
 ---
+
+1. 不出块排查
+
+查看进程是否存活，确保单节点有7个进程
+
+```shell
+$ ps -ef | grep cita- |grep -v grep |wc -l && ps -ef | grep cita- |grep -v grep
```
Comment (leeyr338) [inline docs/cita/ops/exception-handle.md]: `bin/cita top` can replace this command. see:
https://docs.citahub.com/zh-CN/next/cita/getting-started/run-cita#%E9%AA%8C%E8%AF%81-cita-%E6%98%AF%E5%90%A6%E8%BF%90%E8%A1%8C%E6%AD%A3%E5%B8%B8

File: docs/cita/ops/exception-handle.md [hunk 2]
```diff
@@ -2,3 +2,47 @@
...
+cita 6193 6180 0 10:54 ? 00:00:00 cita-jsonrpc -c jsonrpc.toml
+cita 6194 6180 0 10:54 ? 00:00:00 cita-executor -c executor.toml
+cita 6195 6180 0 10:54 ? 00:00:00 cita-chain -c chain.toml
+cita 6202 6180 0 10:54 ? 00:00:00 cita-bft -c consensus.toml -p privkey
+```
+
+查看日志是否有异常输出 （逐个应用排查）
+
+```shell
+$ tail -F test-chain/0/logs/cita-executor.log
```
Comment (leeyr338) [inline docs/cita/ops/exception-handle.md]: It is better to use CITA command, see:
https://docs.citahub.com/zh-CN/next/cita/commands#logs

File: docs/cita/ops/exception-handle.md [hunk 3]
```diff
@@ -2,3 +2,47 @@
...
+```
+
+查看节点间是否互通
+
+```shell
+telnet 192.168.1.103 4000Trying 192.168.1.103...Connected to 192.168.1.103.Escape character is '^]'.
```
Comment (leeyr338) [inline docs/cita/ops/exception-handle.md]: Maybe it is better to use `getPeerInfo` to check networking.

File: docs/cita/ops/service-monitor.md [hunk 4]
```diff
@@ -2,3 +2,8 @@
 id: service-monitor
 title: 服务监控
 ---
+
+1. 主要监控机器指标（CPU、内存、硬盘）使用率，链指标（进程存活、是否出块）开源解决方案 Zabbix、prometheus
```
Comment (leeyr338) [inline docs/cita/ops/service-monitor.md]: 这句话好像不太通顺。你的意思是：
```
主要监控机器指标（CPU、内存、硬盘）使用率，链指标（进程存活、是否出块）
```
?

File: docs/cita/ops/exception-handle.md (hunk 1 above)
Comment (kaikai1024) [reply docs/cita/ops/exception-handle.md]: yes

File: docs/cita/ops/exception-handle.md (hunk 2 above)
Comment (kaikai1024) [reply docs/cita/ops/exception-handle.md]: ok

File: docs/cita/ops/exception-handle.md (hunk 3 above)
Comment (kaikai1024) [reply docs/cita/ops/exception-handle.md]: indeed

File: docs/cita/ops/service-monitor.md (hunk 4 above)
Comment (kaikai1024) [reply docs/cita/ops/service-monitor.md]: I will fix it later.

**Review by leeyr338 — APPROVED** [review]

**Review by nanmuxiusi — APPROVED** [review]

**Pull request merged**